changelog.md

```diff
@@ -1,5 +1,29 @@
 # Connector release changelog
 
+## v1-0.0.5 changelist
+### SDK
+* Avoid inheritance when DefaultAcl is configured to be public
+* Catch errors and runtime exceptions in traverser workers
+* Do not decode + to space in resource names.
+* Add IndexingItemBuilder support for searchQualityMetadata.quality config
+* Parameter: itemMetadata.objectType is deprecated, use itemMetadata.objectType.defaultValue=Item
+
+### Sharepoint
```

Comment:
connector updates may be moved to individual repo.

Reply:
I agree with this, partners may just be using specific reference connectors and not look in the changelog for the SDK. Please remove the connector-specific bits, publish the SDK changelog, and then submit a PR for the specific connectors?